release-1.8/release-notes-draft.md

```diff
@@ -213,6 +213,18 @@ This section contains a list of known issues reported in Kubernetes 1.8 release.
 
 For more information, see [#1996](https://github.com/kubernetes/minikube/issues/1996).
 
+* Audit logs might impact the API server performance and the latency of large request and response calls. The issue is observed under the following conditions: feature gate for advanced audit logging is enabled, audit logging uses the log backend in JSON format, and the audit policy records large events at request and response level.
```

Comment:
nit: note that it's enabled by default: `AdvancedAuditing feature gate is enabled (the default)`